gomod-sync/README.md

```diff
@@ -0,0 +1,101 @@
+# gomod-sync
+
+Utility tool to check and update the Go version specified in the `go.mod` files of all exercises.
+It works by specifying the desired Go version for all the `go.mod` files to be in: the `check` command
```

Comment:
I don't understand the `:` after "to be in".

Reply:
This seems like a copy-paste gone wrong. Thanks for pointing out. Will fix.

Reply:
Turns out that was supposed to be a period `.`.